Sources/Orbit/Support/Components/TimelineSegmentLine.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+struct TimelineSegmentLine: View {
```

Comment:
Is the term TimelineSegmentLine driven by an existing design? In designs, I only found `TimelineLine` component. Those design atomic parts were introduced recently, so even our existing `TimelineStep` is not accurate now and might be renamed in future, but this component seems to already have a name.

Also, as it is internal only, it should not appear in the Timeline.md file

Reply:
Renamed